Dockerfile

```diff
@@ -172,23 +169,42 @@ RUN if [[ -f /docker-context-files/.pypirc ]]; then \
         cp /docker-context-files/.pypirc /root/.pypirc; \
     fi
 
+COPY scripts/docker /scripts/docker
+# fix permission issue in Azure DevOps when running the scripts
+RUN chmod a+x /scripts/docker/*.sh
+
+ARG AIRFLOW_PIP_VERSION
+ENV AIRFLOW_PIP_VERSION=${AIRFLOW_PIP_VERSION}
+
+# Install Airflow ith "--user" flag, so that we can copy the whole .local folder to the final image
+# from the build image and always in non-editable mode
+ENV AIRFLOW_INSTALL_USER_FLAG="--user"
```

Comment:
We don't allow overriding this Env Var, do we? If no, do we really need to have this as a separate env var?

Reply:
It's needed to pass it to the script which is shared between PROD and CI docker and this is what we are passing to the script. ENV Variable is the most "readable" way of doing it. We could pass it by parameters of the script, but I think variables are much easier to maintain.